app/segments/(team)/chooseTeam.js

```diff
@@ -22,62 +30,113 @@ function ChooseTeam() {
       console.log(e);
     }
   }
+
+  const [dialogVisible, setDialogVisible] = useState(false);
+  const hideDialog = () => setDialogVisible(false);
+
   return (
-    <EmptyScreen
-      text={"Choose a team"}
-      postChild={() => (
-        <>
-          <View
-            style={{
-              flexGrow: 1,
-              alignItems: "center",
-              justifyContent: "center",
-            }}
-          >
-            <Button
-              onPress={async () => {
-                await setDoc(doc(db, "teams", "1", "users", currentUserId), {
-                  name: currentUserInfo["displayName"],
-                  // hardcoded pfp string for now, add pfp upload to profile settings in future PR
-                  pfp: "https://cdn.pixabay.com/photo/2015/10/05/22/37/blank-profile-picture-973460_1280.png",
-                  // hardcoded "player" role for now, add role selection to profile settings in future PR
-                  role: "player",
-                  uid: currentUserId,
-                  assigned_data: [],
-                });
-                setCurrentUserId(currentUserId);
-                invalidateMultipleKeys(queryClient, [
-                  ["userInfo", { userId: currentUserId }],
-                ]);
-                router.replace("/");
-              }}
+    <PaperProvider>
+      <EmptyScreen
+        text={"Choose a team"}
+        postChild={() => (
+          <>
+            <View
               style={{
-                backgroundColor: "lightblue",
-                padding: 50,
+                flexGrow: 1,
+                alignItems: "center",
+                justifyContent: "center",
               }}
             >
-              <Text>Join Team</Text>
-            </Button>
-          </View>
-          <View
-            style={{
-              flexGrow: 1,
-              alignItems: "center",
-              justifyContent: "center",
-            }}
-          >
-            <Button
-              onPress={handleSignOut}
+              <Surface
+                style={{
+                  borderRadius: 10,
+                  elevation: 10,
+                }}
+              >
+                <TouchableRipple
+                  onPress={async () => {
+                    const blacklistDoc = await getDoc(
```

Comment:
Can you just check this when this page is opened so you don't have to display this button?